Integrations/ZeroFox/ZeroFox.py

```diff
@@ -0,0 +1,758 @@
+    params: Dict = remove_none_dict(demisto.args())
+    # handle all integer query params
+    for key in ['group', 'label', 'network', 'page', 'policy', 'type']:
+        dict_value_to_integer(params, key)
+    response_content: Dict = list_entities(params)
+    if not response_content:
+        return_outputs('No entities found.', outputs={})
+    elif isinstance(response_content, Dict):
+        entities: List = response_content.get('entities', [])
+        if not entities:
+            return_outputs('No entities found.', outputs={})
+        else:
+            contents: List = [get_entity_contents(entity) for entity in entities]
+            human_readable: List = [get_entity_human_readable_outputs(content) for content in contents]
+            context: Dict = {'ZeroFox.Entity(val.ID && val.ID === obj.ID)': contents}
+            headers: List = ['Name', 'Type', 'Policy', 'Email', 'Tags', 'ID']
+            return_outputs(
+                tableToMarkdown('ZeroFox Entities', human_readable, headers=headers, removeNull=True),
+                context,
+                response_content
+            )
+    else:
+        return_outputs('No entities found.', outputs={})
+
+
+def fetch_incidents():
+    date_format = '%Y-%m-%dT%H:%M:%S'
+    last_run = demisto.getLastRun()
+    if last_run and last_run.get('last_fetched_event_timestamp'):
+        last_update_time = last_run['last_fetched_event_timestamp']
+    else:
+        last_update_time = parse_date_range(FETCH_TIME, date_format=date_format)[0]
+    incidents = []
+    limit: int = int(demisto.params().get('fetch_limit', ''))
+    response_content = list_alerts({'sort_direction': 'asc', 'limit': limit, 'min_timestamp': last_update_time})
+    alerts: List = response_content.get('alerts', [])
+    if alerts:
+        for alert in alerts:
+            incident = alert_to_incident(alert)
+            incidents.append(incident)
+        # max_update_time is the timestamp of the last alert in alerts (alerts is a sorted list)
+        last_alert_timestamp: str = str(alerts[-1].get('timestamp', ''))
+        # ZeroFox is using full timestamp ISO 8061 format which includes the GMT field i.e (+00:00/-00:00)
+        # The option to refine the search of alerts in the API is by the ISO 8061 format without the GMT field.
+        if '+' in last_alert_timestamp:
+            max_update_time = last_alert_timestamp.split('+')[0]
+        else:
+            max_update_time = last_alert_timestamp.split('-')[0]
+        # add 1 second to last alert timestamp, in order to prevent duplicated alerts
+        max_update_time = (datetime.strptime(max_update_time, date_format) + timedelta(seconds=1)).isoformat()
+        max_update_time = last_update_time
+        demisto.setLastRun({'last_fetched_event_timestamp': max_update_time})
+    demisto.incidents(incidents)
+
+
+def test_module():
+    """
+    Performs basic get request to get item samples
+    """
+    get_authorization_token()
```

Comment:
this is already called from `http_request` which is called from `get_policy_types` so you can remove it from here